src/components/v5/common/ActionSidebar/partials/ManageColonyGroup/GroupList.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  ArrowUpRight,
+  Coins,
+  LockSimpleOpen,
+  CoinVertical,
+  UsersThree,
+  Pencil,
+  UserSwitch,
+  IdentificationCard,
+  HouseLine,
+  Upload,
+  UserCircleCheck,
+  TrafficCone,
+} from '@phosphor-icons/react';
+
+import { Action } from '~constants/actions.ts';
+import { formatText } from '~utils/intl.ts';
+
+export const GROUP_FUNDS_LIST = [
+  {
+    title: formatText({ id: 'actions.transferFunds' }),
+    description: formatText({
+      id: 'actions.description.transferFunds',
+    }),
+    Icon: ArrowUpRight,
+    action: Action.TransferFunds,
+  },
+  {
+    title: formatText({ id: 'actions.mintTokens' }),
+    description: formatText({
+      id: 'actions.description.mintTokens',
+    }),
+    Icon: Coins,
+    action: Action.MintTokens,
+  },
+  {
+    title: formatText({ id: 'actions.unlockToken' }),
+    description: formatText({
+      id: 'actions.description.unlockToken',
+    }),
+    Icon: LockSimpleOpen,
+    action: Action.UnlockToken,
+  },
+  {
+    title: formatText({ id: 'actions.manageTokens' }),
+    description: formatText({
+      id: 'actions.description.manageTokens',
+    }),
+    Icon: CoinVertical,
+    action: Action.ManageTokens,
+  },
+];
+
+export const GROUP_TEAMS_LIST = [
+  {
+    title: formatText({ id: 'actions.createNewTeam' }),
+    description: formatText({
+      id: 'actions.description.createNewTeam',
+    }),
+    Icon: UsersThree,
+    action: Action.CreateNewTeam,
+  },
+  {
+    title: formatText({ id: 'actions.editExistingTeam' }),
+    description: formatText({
+      id: 'actions.description.editExistingTeam',
+    }),
+    Icon: Pencil,
+    action: Action.EditExistingTeam,
+  },
+];
+
+export const GROUP_ADMIN_LIST = [
+  {
+    title: formatText({ id: 'actions.manageReputation' }),
+    description: formatText({
+      id: 'actions.description.manageReputation',
+    }),
+    Icon: UserSwitch,
+    action: Action.ManageReputation,
+  },
+  {
+    title: formatText({ id: 'actions.managePermissions' }),
+    description: formatText({
+      id: 'actions.description.managePermissions',
+    }),
+    Icon: IdentificationCard,
+    action: Action.ManagePermissions,
+  },
+  {
+    title: formatText({ id: 'actions.editColonyDetails' }),
+    description: formatText({
+      id: 'actions.description.editColonyDetails',
```

Comment:
On Figma, there seems to be a slight difference between the format of the word "colony" like on Desktop, the word is lowercased but on mobile, the letter "c" is capitalised. I know it's such a minor thing but please liaise with @melyndav to see what's what

**Figma desktop**
https://www.figma.com/design/uKxVl441Wnn02SNiOA24De/Navigation?node-id=4537-58457&t=apyC4gKEtDzqfoo7-4
![image](https://github.com/user-attachments/assets/d3e2c100-7b2f-41a5-a8c0-0f0e4f3b0c1d)

**Figma mobile**
https://www.figma.com/design/uKxVl441Wnn02SNiOA24De/Navigation?node-id=4537-60024&t=apyC4gKEtDzqfoo7-4

![image](https://github.com/user-attachments/assets/f86149a2-6db8-4f1b-a428-b0758dde86a9)

Reply:
Just got confirmation that word "colony" should have a lowercase letter "c" @Nortsova 👌